src/plugins/share/public/components/url_panel_content.tsx

```diff
@@ -190,13 +207,25 @@ export class UrlPanelContent extends Component<Props, State> {
     return url;
   };
 
-  private makeUrlEmbeddable = (url: string) => {
-    const embedQueryParam = '?embed=true';
+  private getEmbedQueryParams = (): string => {
+    return [
+      ['&show-top-menu=true', this.state.urlParamsSelectedMap.topMenu],
```

Comment:
These magic strings are used here and in the dashboard plugin - to make sure they stay the same, please put them in constants, export them from the root level `index.ts` file of the share plugin and import them in the dashboard plugin.